app/controllers/peeps.rb

```diff
@@ -0,0 +1,7 @@
+post '/peeps' do
+  content = params["content"]
+  author = params["author"]
+  datetime = DateTime.now
```

Comment:
You can do this automatically in datamapper http://datamapper.org/docs/dm_more/timestamps.html